templates/common/infra/bicep/core/host/container-apps-environment.bicep

```diff
@@ -16,12 +18,17 @@ resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2022-03-01'
         sharedKey: logAnalyticsWorkspace.listKeys().primarySharedKey
       }
     }
+    daprAIInstrumentationKey: daprEnabled && applicationInsightsName != '' ? applicationInsights.properties.InstrumentationKey : ''
```

Comment:
Is this a container apps thing to have a special prop for dapr instrumentation?  Seems odd that dapr would be in the name and not use a generic instrumentation key name.

Reply:
yes, there is indeed a special prop for dapr instrumentation on the managedEnvironment resource.